agentforce/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,144 @@
+= Agentforce Connector 1.0 - Mule 4
+:lc-connector-name: agentforce
+
+Anypoint Connector for Agentforce (Agentforce Connector) provides connectivity to the https://help.salesforce.com/s/articleView?id=sf.generative_ai_trust_layer.htm&type=5[Salesforce Einstein Trust Layer]. 
+
+Agentforce Connector offers a range of advanced features that simplify and enhance AI integration in MuleSoft applications:
+
+Use this connector to:
+
+* Build AI Agents in a no-code environment
+* Leverage existing investments (APIs, integrations, and templates) as tooling for the AI agents.
+* Manage all Large Language Model (LLM) interactions through the Salesforce Einstein Trust Layer
+* Access AI services, RAG, tools, chain, and more
+* Make use of external vector stores
+
+For information about compatibility and fixed issues, see the Agentforce Connector release notes.
+
+== Before You Begin
+
+To use this connector, you must be familiar with:
+
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* How to create a Mule app using Anypoint Code Builder or Anypoint Studio
+
+Before creating an app, you must have:
+
+* Java 8 or 17
+* Apache Maven
+* Credentials to access the Agentforce Connector target resource
+* Anypoint Platform
+* The latest versions of Anypoint Code Builder or Anypoint Studio
+
+== Common Use Cases for the Connector
```

Comment:
@valkyrie69 I know this is how this section was organized previously but I'm wondering if we should consider alternatives...especially knowing that we'll be adding support for new operations